scss/5-objects/_prose.scss

```diff
@@ -70,6 +66,39 @@
   }
 }
 
+/**
+ * Prose placeholder, allows components to extend prose styles
+ * without using a mixin or extending a raw class name.
+ */
+%cads-prose {
+  @include cads-prose;
+}
+
+/**
+ * Prose styles (cascading)
+ *
+ * Apply this class to blocks of HTML that you want to give consistent typographic styles,
+ * typically this is applied to blocks of copy that come from a content management system
+ * or local file.
+ */
+.cads-prose {
+  @include cads-prose;
+}
+
+/**
+ * Prose styles (scoped)
+ *
+ * This applies the same styles as the default cads-prose class but scoped using
+ * a child combinator selector to only apply the styles to immediate children.
+ * This is useful when you are embedding lots of rich components within a prose
+ * block and you want to more narrowly scope the effects of the cascade.
+ */
+.cads-prose-scoped {
```

Comment:
Nitpick so feel free to ignore - wonder if it's worth having a more descriptive name like one of these terrible options, but better:
- `cads-prose-on-children`
- `cads-prose-scoped-to-children`
or smth

Reply:
LOL 👏🏻